utils/mtpoe-ctrl/Makefile

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (C) 2018 adron@yapic.net
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mtpoe-ctrl
+PKG_SOURCE_VERSION:=9c42b74274cca10810363dbbb6d161bfb9cedb7e
+PKG_MIRROR_HASH:=1ac2902161ba4bb1e8af397aa48821ccea95740199b1c155f98161c7ff3a93df
+PKG_VERSION:=1.15
+PKG_RELEASE:=18
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/prudy/mtpoe_ctrl.git
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Sergey Sergeev <adron@yapic.net>
+PKG_LICENSE:=GPL-2.0
```

Comment:
SPDX license identifier is deprecated. You need to check the latest one.

Reply:
Will check.

Reply:
Well, the original repo states:
`Licence is GNU GPL V3`

Reply:
Yes, but not in here https://github.com/adron-s/mtpoe_ctrl/blob/master/openwrt/package/utils/mtpoe_ctrl/Makefile

Reply:
I think that the license information in the source code repository is the only valid info, the one in OpenWrt package description is more for automated SBOM generation than anything else.

It cannot be dual licensed if the repo is GPL v3

Reply:
Can you then take mtpoe code as part of packages at all? I doubt it.